std_execution.bs

```diff
@@ -4014,7 +4014,9 @@ from the operation before the operation completes.
 
 2. A `run_loop` instance has an associated <i>count</i> that corresponds to the number of work items that are in its queue. Additionally, a `run_loop` has an associated <i>state</i> that can be either <i>finishing</i> or <i>not-finishing</i>.
 
-3. [<i>Note:</i> Implementations are encouraged to use an intrusive queue of operation states to hold the work units to make scheduling allocation-free. -- <i>end note</i>]
+3. Concurrent invocations of the member functions of `run_loop`, other than its destructor, do not introduce data races. The member functions `pop_front`, `push_back`, and `finish` execute atomically.
```

Comment:
Maybe we should exclude the `run()` member from the comment about data races since we don't mean for it to be called concurrently.

Reply:
Good point.